### PR TITLE
Issue #28548: Invoice improvements to customer shipvia handling.

### PR DESCRIPTION
**Pull request by andersdd**

### CODE REVIEW

File: guiclient/invoice.ui [hunk 1]
```diff
@@ -1426,11 +1426,10 @@ to be bound by its terms.</comment>
   </customwidget>
  </customwidgets>
  <tabstops>
+  <tabstop>_invoiceDate</tabstop>
```
Comment (gilmoskowitz) [inline guiclient/invoice.ui]: Please revert the changes to tab order unless there's been a specific request to move `_invoiceDate` to the front.

**Comment by andersdd** [comment]
The tabstop change was done my me, as the screen was starting at invoice date anyway and then bouncing around other widgets.  This order seems more logical.
